docs/input/sparkplug-b-input.md

```
@@ -156,14 +156,14 @@ Here's how a Sparkplug B message maps to UMH-Core using the Modified Parris Meth
 | Field | Type | Default | Description |
 |-------|------|---------|-------------|
 | `identity.group_id` | `string` | **required** | Sparkplug B Group ID |
-| `identity.edge_node_id` | `string` | `""` | Required only for `primary` mode (used as host_id for STATE topic) |
+| `identity.edge_node_id` | `string` | `""` | Required only for `primary` role (used as host_id for STATE topic) |
 
-### Mode Section
+### Role Section
 | Field | Type | Default | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)